### PR TITLE
Create ui-c8y-1021-24-1-allow-device-type-dashboards-to-work-for-device-types-containing-a.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61863](https://cumulocity.atlassian.net/browse/MTM-61863)
Original PR: [7774](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7774)

[MTM-61863]: https://cumulocity.atlassian.net/browse/MTM-61863?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-24-1-allow-device-type-dashboards-to-work-for-device-types-containing-a.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: allow device type dashboards to work for device types containing a dot (#7774)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-24-1-allow-device-type-dashboards-to-work-for-device-types-containing-a.md]: The Claude AI suggested to use the following title:
```suggestion
title: Device type dashboards can now be used for device types containing a dot
```

File: content/change-logs/application-enablement/ui-c8y-1021-24-1-allow-device-type-dashboards-to-work-for-device-types-containing-a.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61863
+version: 1021.24.1
+---
+allow device type dashboards to work for device types containing a dot (#7774)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-24-1-allow-device-type-dashboards-to-work-for-device-types-containing-a.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, device type dashboards did not work properly for device types that contained a dot (".") in their name. This has now been fixed. Device type dashboards can be used for all device types, regardless of whether their name contains special characters like dots. This change ensures that users can properly monitor and visualize data for all their device types, including those with names containing dots, enabling them to gain insights into the performance and behavior of these devices.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-24-1-allow-device-type-dashboards-to-work-for-device-types-containing-a.md (hunk 2 above)
Comment (reey) [reply content/change-logs/application-enablement/ui-c8y-1021-24-1-allow-device-type-dashboards-to-work-for-device-types-containing-a.md]:
```suggestion
Previously, device type dashboards did not work properly for devices that contained a dot (".") in their type. This has now been fixed.
```

**Review by reey — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**